client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content.tsx

```diff
@@ -199,6 +199,39 @@ export default function SiteStatusContent( {
 			content = <Gridicon icon="minus-small" size={ 18 } className="sites-overview__icon-active" />;
 			break;
 		}
+		case 'up': {
```

Comment:
I feel like we should have a different component to handle these as this component is getting bigger.

We can do how it is done for Monitor and do an early return.

Reply:
I gave this a shot. Let me know what you think.

Reply:
I have left a comment